app/src/main/java/fr/paug/androidmakers/ui/activity/DetailActivity.java

```diff
@@ -126,6 +126,9 @@ public void onChipClick(View view) {
                 final DetailViewSpeakerInfoElementBinding speakerInfoElementBinding = DataBindingUtil.inflate(getLayoutInflater(), R.layout.detail_view_speaker_info_element, null, false);
                 speakerInfoElementBinding.speakerBio.setMovementMethod(LinkMovementMethod.getInstance());
                 speakerInfoElementBinding.setSpeaker(speaker);
+                if (speaker.ribbonList != null && !speaker.ribbonList.isEmpty()) {
```

Comment:
Note to reviewer: we can add (in this PR or later) a way to sort ribbons.

Reply:
We should have a method in the speaker object to get the ribbon. If we do that, we can remove this code and directly use it directly via the speaker in the XML with databinding.

What do you think ?

Reply:
Ok, will try to do it tonight.